docs/guides/server/bootstrap-admin-recovery.adoc

```diff
@@ -0,0 +1,90 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Admin bootstrap and recovery"
+summary="Learn how to bootstrap and recover admin account." >
```

Comment:
As I mentioned during the meeting, initially I tried: 
```suggestion
summary="Learn how to bootstrap and recover admin account." 
includedOptions="bootstrap-admin-*">
```
but the Relevant options table ended up to be empty. Any idea why?